tcms/report/data.py

```diff
@@ -37,7 +36,7 @@
 
 
 def models_to_pks(models):
-    return [model.pk for model in models]
+    return [model.pk for model in models]  # pylint: disable=avoid-list-comprehensions
```

Comment:
`avoid-list-comprehension`, without an `s` at the end. Also see your pylint log, it tells you there's an error with this option.